dataactvalidator/scripts/readZips.py

```diff
@@ -247,7 +271,10 @@ def read_zips():
             citystate_file = os.path.join(CONFIG_BROKER["path"], "dataactvalidator", "config", "ctystate.txt")
             parse_citystate_file(open(citystate_file), sess)
 
+            census_file = os.path.join(base_path, "census_congressional_districts.csv")
+
         update_state_congr_table(sess)
+        update_state_congr_table_census(census_file, sess)
```

Comment:
Does a second function make sense here or does it make more sense to make both of them part of the `update_state_congr_table` since that function name seems to encompass what is happening here as well.

Reply:
I kept it separate so that it would not get lost in `update_state_congr_table`, since the census info comes from a file and the while `update_state_congr_table` is based on querying the Zips table. Was your idea that the all the code

Reply:
Maybe we should rename the first function in that case to make it less generic. Something like `update_state_congr_table_current`? Since right now the name makes it seem like all the updates should happen inside it.